data_nutch_data_nci/conf/nutch-site.xml

```diff
@@ -84,56 +88,61 @@
 	</property> 
 
 	<!-- If fetcher.threads.per.queue == 1 then comment the lines below -->
-	<property>
+	<!--<property>
 	  <name>fetcher.server.min.delay</name>
 	  <value>0.3</value>
 	  <description>NCI OVERRIDE: We need to ignore crawl delays in robots.txt for many Drupal sites which are set at 10 sec.  Setting too many threads will kill sites.  This setting acts like fetcher.server.delay, but </description>
 	</property>
 
+	-->
```

Comment:
Should this instead be deleted?  Or was this a temporary change that didn't get reverted?

Reply:
Keeping this comment is deliberate.